ddp-workspace/projects/toolkit/src/lib/components/privacy-policy/prionPrivacyPolicy.component.ts

```diff
@@ -0,0 +1,243 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'prion-privacy-policy',
+  template: `
+    <section class="PageContent-section">
```

Comment:
For the record, if I had more time, I'd go back and make this privacy policy into an activity, but I hadn't thought of that at the time and I don't think we have time now.